prism/src/main/java/org/apache/falcon/handler/SLAMonitoringHandler.java

```diff
@@ -60,7 +60,8 @@ private void updateSLAMonitoring(String clusterName, String[] outputFeedNamesLis
                 String templatePath = new Path(storage.getUriTemplate(LocationType.DATA)).toUri().getPath();
                 Date date = FeedHelper.getDate(templatePath, new Path(outputFeedInstancePathsList[index]),
                     EntityUtil.getTimeZone(feed));
-                FeedSLAMonitoringService.get().makeFeedInstanceAvailable(outputFeedNamesList[index], clusterName, date);
+                EntitySLAMonitoringService.get().makeFeedInstanceAvailable(outputFeedNamesList[index],
+                        clusterName, date);
```

Comment:
Update this class to invoke makeProcessInstanceAvailable (you'll need to add this method) when a process completes successfully.